papertty.py

```diff
@@ -537,6 +582,73 @@ def sigusr1_handler(sig, frame):
                 oldimage = None
                 oldbuff = ''
                 flags['scrub_requested'] = False
+
+            if flags['show_menu']:
+                flags['show_menu'] = False
+                print()
+                print('Rendering paused. Enter')
+                print('    (f) to change font,')
+                print('    (s) to change spacing,')
+                if ptty.is_truetype:
+                    print('    (h) to change font size,')
+                print('    (x) to exit,')
+                print('    anything else to continue.')
+                print('Use --font {} --size {} --spacing {} for the current setting.'.format(ptty.fontfile, ptty.fontsize, ptty.spacing))
```

Comment:
Minor readability tweak: personally I find having the arguments in the middle of text a bit cumbersome for copy-pasting etc. so moving them to a separate line, flush with the other sub-items seems more readable.

I'd also adjust the whole menu printing a bit (repeated `print` statements, while simple, are a bit of an eyesore), but it can be done if/when the menu gets more populated.

```suggestion
                print('Command line arguments for current settings:\n    --font {} --size {} --spacing {}'.format(ptty.fontfile, ptty.fontsize, ptty.spacing))
```

Reply:
Valid point.